src/module/script.js

```diff
@@ -0,0 +1,57 @@
+const ul = document.getElementById('todo_list');
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+export const getToDos = () => todos;
+
+export const addToDo = (todo) => {
+  const txtInput = {
```

Comment:
- [ OPTIONAL ] Having a descriptive and meaningful variable names is part of best practice as software developer. What do you think if you can refactor the name of this variable to something like `todo`, `task` etc?